app/apis/ims/__init__.py

```diff
@@ -0,0 +1,80 @@
+    api.add_resource(lti_11.Lti, evaluate_url('/lti/11/launch', prefix_url))
+
+    api.add_resource(cim.CIMRequest, evaluate_url('/contentItem/request', prefix_url))
+    api.add_resource(cim.ContentItemData, evaluate_url('/contentItem/data', prefix_url))
+
+    api.add_resource(lti_13.Lti13Launch, evaluate_url('/lti/13/launch', prefix_url))
+    api.add_resource(lti_13.JWTPayloadData, evaluate_url('/lti/13/payloadData', prefix_url))
+    api.add_resource(lti_13.Login, evaluate_url('/lti/13/login', prefix_url))
+
+    api.add_resource(deep_link.DeepLinkingPayloadData, evaluate_url('/deepLink/payloadData', prefix_url))
+    api.add_resource(deep_link.DeepLinkingContent, evaluate_url('/deepLink/content', prefix_url))
+    api.add_resource(deep_link.DeepLinkingOptions, evaluate_url('/deepLink/options', prefix_url))
+    api.add_resource(deep_link.DeepLinkingFrame, evaluate_url('/deepLink/frame', prefix_url))
+
+    api.add_resource(proctoring.ProctoringPayloadData, evaluate_url('/proctoring/payloadData', prefix_url))
+    api.add_resource(proctoring.BuildProctoringStartReturnPayload,
+                     evaluate_url('/buildProctoringStartReturnPayload', prefix_url))
+    api.add_resource(proctoring.BuildProctoringEndReturnPayload,
+                     evaluate_url('/buildProctoringEndReturnPayload', prefix_url))
+
+    api.add_resource(names_roles.NamesAndRoles, evaluate_url('/namesAndRoles', prefix_url))
+    api.add_resource(names_roles.NamesAndRoles2, evaluate_url('/namesAndRoles2', prefix_url))
+    api.add_resource(names_roles.NamesRolesPayloadData, evaluate_url('/namesAndRoles/payloadData', prefix_url))
+
+    api.add_resource(groups.Groups, evaluate_url('/groups', prefix_url), evaluate_url('/getgroups', prefix_url))
+    api.add_resource(groups.GroupsPayloadData, evaluate_url('/groups/payloadData', prefix_url))
+    api.add_resource(groups.GroupSets, evaluate_url('/groups/sets', prefix_url))
+    api.add_resource(groups.GroupSetsPayloadData, evaluate_url('/groups/sets/payloadData', prefix_url))
+
+    api.add_resource(assignment_grades.AssignmentAndGrades, evaluate_url('/assignAndGrades', prefix_url))
+    api.add_resource(assignment_grades.ReadColumn, evaluate_url('/agsReadCols', prefix_url))
+    api.add_resource(assignment_grades.AddColumn, evaluate_url('/agsAddcol', prefix_url))
+    api.add_resource(assignment_grades.DeleteColumn, evaluate_url('/agsDeleteCol', prefix_url))
+    api.add_resource(assignment_grades.Results, evaluate_url('/agsResults', prefix_url))
+    api.add_resource(assignment_grades.Scores, evaluate_url('/agsScores', prefix_url))
+    api.add_resource(assignment_grades.ClearScores, evaluate_url('/agsClearScores', prefix_url))
+    api.add_resource(assignment_grades.SubmitAttempt, evaluate_url('/agsSubmitAttempt', prefix_url))
+    api.add_resource(assignment_grades.AssignmentGradesPayloadData, evaluate_url('/agPayloadData', prefix_url))
+
```

Comment:
The AGS URL pattern is different than the others. Is there a reason you with withe /agsReadCols instead of /assignmentAndGrades/readCols, etc?

Reply:
Originally I created all this paths using the same as the node tool, but I'm changing them while we're developing the endpoints. That's the only reason.